### PR TITLE
Ukrainian translation of the blog post "Using Go workspaces in Kubernetes"

### PR DESCRIPTION
**Pull request by Andygol**

### CODE REVIEW

File: content/uk/blog/_posts/2024-03-19-go-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: 'Використання робочих просторів Go в Kubernetes'
+date: 2024-03-19T08:30:00-08:00
+slug: go-workspaces-in-kubernetes
+canonicalUrl: https://www.kubernetes.dev/blog/2024/03/19/go-workspaces-in-kubernetes/
```
Comment (sftim) [inline content/uk/blog/_posts/2024-03-19-go-workspaces.md:6]: Ah, for this page don't include the `canonicalUrl`:
```suggestion
```

**Comment by sftim** [comment]
/hold

The original article has disappeared.

**Comment by natalisucks** [comment]
As this article is on the contributor site, and no longer on the main Kubernetes blog, I believe this blog should not be translated to this location, given the contributor site doesn't have a Ukrainian version.

@Andygol there are many other blogs from previous releases that can be translated for the purposes of the Ukrainian localization. I suggest to start with the former release, as this could be relevant to our Ukrainian readers.

**Comment by Andygol** [comment]
@natalisucks, thank you for clarifying this issue. Of course, I'll be translating other blog posts as soon as I get the chance. As for this translation, is there a way to move it to the appropriate location so that my translation work doesn't go to waste?

**Comment by natalisucks** [comment]
@Andygol Unfortunately, not. The contributor site lives at [kubernetes.dev](https://www.kubernetes.dev), and the blog is at [kubernetes.dev/blog/](https://www.kubernetes.dev/blog/), neither of which have any versions in any other language. We preface all translations to be concentrated on the main Kubernetes documentation first, and due to an error regarding blog publication and current release team work, the blog you chose to publish was an outlier and removed.

To avoid this kind of thing happening in the future, my suggestion is to translate blogs on the main documentation from a prior release, or if they're not related to release-based work, checking in with the blog team at #sig-docs-blog on the Kubernetes Slack to clarify, if needed. The Release Team, with @sftim, is currently working on adding another label to make it clearer which blogs are associated with a release, so that more contributors have this information.

**Comment by natalisucks** [comment]
As this blog post is no longer on the main Kubernetes blog, but living on the [Contributor Site](https://www.kubernetes.dev/blog/2024/03/19/go-workspaces-in-kubernetes/), I am closing this PR.

/close

**Pull request closed without merging**